source/jormungandr/jormungandr/interfaces/v1/serializer/api.py

```diff
@@ -34,15 +34,44 @@
 from jormungandr.interfaces.v1.serializer import pt, schedule, report, base, status, geo_status, graphical_isochron, \
     heat_map
 from jormungandr.interfaces.v1.serializer.base import NullableDictSerializer, LambdaField, PbNestedSerializer, \
-    DescribedField
+    DescribedField, AmountSerializer
 from jormungandr.interfaces.v1.serializer.fields import ErrorSerializer, FeedPublisherSerializer, \
         PaginationSerializer, LinkSchema, NoteSerializer, ExceptionSerializer
 from jormungandr.interfaces.v1.make_links import create_external_link
-from jormungandr.interfaces.v1.serializer.journey import TicketSerializer, ContextSerializer, JourneySerializer
+from jormungandr.interfaces.v1.serializer.journey import TicketSerializer, JourneySerializer
 import serpy
 
 from jormungandr.interfaces.v1.serializer.jsonschema.fields import Field, MethodField
 from jormungandr.interfaces.v1.serializer.time import DateTimeDictField
+from jormungandr.utils import get_current_datetime_str, get_timezone_str
+
+
+class CO2Serializer(PbNestedSerializer):
+    co2_emission = AmountSerializer(attr='car_co2_emission', display_none=False)
+
+
+class ContextSerializer(PbNestedSerializer):
+    car_direct_path = MethodField(schema_type=CO2Serializer(), display_none=False)
+    current_datetime = MethodField(schema_type=str,
+                                   display_none=False,
+                                   description='The datetime considered as "now". Used for debug, default is '
```

Comment:
Please change it to: The datetime of the request (considered as "now").
The rest of it is unrelated.

Reply:
Done